eos/utils/Makefile.am

```diff
@@ -22,8 +22,13 @@ libeosutils_la_SOURCES = \
 	diagnostics.cc diagnostics.hh \
 	exception.cc exception.hh \
 	expression.cc expression.hh expression-fwd.hh \
+	expression-cacher.hh \
+	expression-cloner.hh \
+	expression-evaluator.hh \
+	expression-maker.hh \
 	expression-observable.cc expression-observable.hh \
-	expression-parser.hh expression-parser-impl.hh \
+	expression-parser.cc expressions-parser.hh expression-parser-impl.hh \
```

Comment:
@dvandyk you changed my correct line and added a faulty `s` ^^

Reply:
Uff, I rebased onto what was already in gh/issue359. Maybe you did not commit and force push?

Reply:
Ok, it is corrected :)